provider-service/vai/internal/provider/job_enricher_test.go

```diff
@@ -23,52 +25,74 @@ func (m *MockPipelineSchemaHandler) extract(raw map[string]any) (*PipelineValues
 }
 
 var _ = Describe("DefaultJobEnricher", func() {
+	Context("Enrich", Ordered, func() {
+		var (
+			pipelineSchemeHandler MockPipelineSchemaHandler
+			labelSanitizer        mocks.MockLabelSanitizer
+			defaultJobEnricher    DefaultJobEnricher
+		)
+
+		pipelineValues := PipelineValues{
+			name: "enriched",
+			labels: map[string]string{
+				"key":              "value",
+				"pipeline-version": "0.0.1",
+				"schema_version":   "2.1.0",
+				"sdk_version":      "kfp-2.12.2",
+				"Other&%":          "someVAl!ue",
+			},
+			pipelineSpec: &structpb.Struct{},
+		}
+
+		BeforeEach(func() {
+			pipelineSchemeHandler = MockPipelineSchemaHandler{}
```

Comment:
Think this should be `schema` not `scheme` - same in other places below
```suggestion
			pipelineSchemaHandler = MockPipelineSchemaHandler{}
```